### PR TITLE
added ability to submit

### PR DESCRIPTION
**Pull request by xivh**
updated get-access-token to accept username/password for authentication that requires a user
added submit as an example (there are more available endpoints)
no error handling
didn't update tests

### CODE REVIEW

File: src/creddit/client.clj [hunk 1]
```diff
@@ -218,3 +242,29 @@
   [credentials names]
   (-> (http-get credentials (str "https://www.reddit.com/by_id/" (string/join "," names) "/.json"))
       (parse-response)))
+
+; other functions use credentials directly, unsure why mine needs to get them from the client with get-in...
+(defn submit
+  [credentials subreddit kind title content]
+  (-> (client/post "https://oauth.reddit.com/api/submit"
+                   (cond (= kind "link")
```
Comment (ThatGuyHughesy) [inline src/creddit/client.clj:250]: The value of the two cond expressions are basically identical except for the value of `:kind` and the keys `:url` or `:text`.

`kind` is being passed in so we can say `{:kind kind}` as for `:url` and `:text` I think we could use a `let` and then either a `cond` or `merge` to add they correct key? What do you think?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
```
Comment (ThatGuyHughesy) [inline .github/stale.yml:1]: Not sure why this is considered an addition as this file already exists in master and is identical?

Comment (xivh) [reply .github/stale.yml:1]: Not sure, I didn't edit that file.

Comment (ThatGuyHughesy) [reply .github/stale.yml:1]: I checked your fork and your `submit` branch is 2 commits behind (the commit which added this file) so you'll need to rebase.

File: src/creddit/client.clj (hunk 1 above)
Comment (xivh) [reply src/creddit/client.clj]: I updated it.

**Pull request merged**